resources/lang/en.json

```diff
@@ -96,6 +96,9 @@
   "All {{assets}} will be included in this export.": "All {{assets}} will be included in this export.",
   "All assignments were saved.": "All assignments were saved.",
   "All Cases": "All Cases",
+  "All cases": "All Cases",
+  "all cases": "All Cases",
+  "ALL CASES": "All Cases",
```

Comment:
The translated string should respect the word capitalization,  i. e.  ALL CASES--> ALL CASES, etc.